src/modality-specific-files/physiological-recordings.md

```diff
@@ -14,41 +9,167 @@ JSON file for storing metadata fields (see below).
       -   [`7t_trt`](https://github.com/bids-standard/bids-examples/tree/master/7t_trt)
       -   [`ds210`](https://github.com/bids-standard/bids-examples/tree/master/ds210)
 
-Template:
+## General specifications
 
-```Text
-sub-<label>/[ses-<label>/]
-    <datatype>/
-        <matches>[_recording-<label>]_physio.tsv.gz
-        <matches>[_recording-<label>]_physio.json
-```
+Continuous (that is, regularly sampled over time at a fixed frequency)
+physiological recordings such as cardiac and respiratory signals, and
+asynchronous events corresponding to those signals MAY be specified using
+[compressed tabular files](../common-principles.md#compressed-tabular-files)
+([TSV.GZ file](../glossary.md#tsvgz-extensions)).
+TSV.GZ files MUST be accompanied by a JSON file with the same name as their
+corresponding tabular file but with a `.json` extension.
 
-For the template directory name, `<datatype>` can correspond to any data
-recording modality, for example `func`, `anat`, `dwi`, `meg`, `eeg`, `ieeg`,
-or `beh`.
+!!! warning "Caution"
 
-In the template filenames, the `<matches>` part corresponds to task filename
-before the suffix.
-For example for the file `sub-control01_task-nback_run-1_bold.nii.gz`,
-`<matches>` would correspond to `sub-control01_task-nback_run-1`.
+    Columns of TSV.GZ files MUST be defined in the corresponding JSON sidecar
+    and the tabular content MUST NOT include a header line.
+
+Summary template:
+
+<div class="highlight">
+<pre><code><a href="../appendices/entities.html#sub">sub-&lt;label&gt;</a>/
+    [<a href="../appendices/entities.html#ses">ses-&lt;label&gt;</a>/]
+        &lt;<a href="../glossary.html#data_type-common_principles">datatype</a>&gt;/
+            &lt;matches&gt;[_<a href="../appendices/entities.html#recording">recording</a>-&lt;<a href="../glossary.html#label-common_principles">label</a>&gt;]_&lt;<a href="../glossary.html#physio-suffixes">physio</a>|<a href="../glossary.html#physioevents-suffixes">physioevents</a>&gt;<a href="../glossary.html#json-extensions">.json</a>
+            &lt;matches&gt;[_<a href="../appendices/entities.html#recording">recording</a>-&lt;<a href="../glossary.html#label-common_principles">label</a>&gt;]_&lt;<a href="../glossary.html#physio-suffixes">physio</a>|<a href="../glossary.html#physioevents-suffixes">physioevents</a>&gt;<a href="../glossary.html#tsvgz-extensions">.tsv.gz</a>
+</code></pre></div>
```

Comment:
AFAIK shouldn't be HTML here -- I can't read it... Use MACRO as below or demonstrate on a simple `Text` example?

Reply:
Problem is that that `matches` is not in the glossary so we can't have macro yet.

Opened this so we can have it in the glossary and then after make it easier to MACRO the :poop: out of this.
https://github.com/bids-standard/bids-specification/pull/1781

Reply:
See also: https://github.com/bids-standard/bids-specification/pull/1128#discussion_r1536958111

Reply:
and also... all PRs are rendered as HTML: https://bids-specification--1128.org.readthedocs.build/en/1128/modality-specific-files/physiological-recordings.html#physiology-events

😉

Reply:
> AFAIK shouldn't be HTML here -- I can't read it... Use MACRO as below or demonstrate on a simple `Text` example?

Yes, we're using this to set some expectation for the macros to deal with this.